build-java/action.yml

```diff
@@ -95,15 +101,19 @@ runs:
       env:
         AWS_ACCESS_KEY_ID: ${{ inputs.mavenS3AccessKeyId }}
         AWS_SECRET_ACCESS_KEY: ${{ inputs.mavenS3SecretAccessKey }}
+        GITHUB_PACKAGES_USERNAME: ${{ inputs.tokenUsername }}
+        GITHUB_PACKAGES_TOKEN: ${{ inputs.token }}
       shell: bash
-      run: mvn -Dmaven.wagon.http.pool=false -s settings.xml -B clean deploy
+      run: mvn -Dmaven.wagon.http.pool=false -s settings.xml -B clean verify
```

Comment:
common-service repo will need "deploy"

Reply:
`common-service` doesn't use this. See https://github.com/cartoncloud/common-service/blob/main/.github/workflows/publish-release.yml